README.md

```diff
@@ -57,6 +57,12 @@ Supporting developers is an important task for webpack CLI. Thus, webpack CLI pr
 -   [`webpack-cli generate-loader`](./packages/generate-loader/README.md#webpack-cli-generate-loader) - Initiate new loader project.
 -   [`webpack-cli serve`](./packages/serve/README.md#webpack-cli-serve) - Use webpack with a development server that provides live reloading.
 
+> Deprecated commands as of v3.3.3
```

Comment:
Deprication refers that these commands will be removed subsequent versions. But as these commands have been removed already w/o a deprication notice, we must write something like `Removed Commands` IMO

Reply:
Depreciated is ok

Reply:
If they have been removed, they are removed, not deprecated.